a3p-integration/package.json

```diff
@@ -4,9 +4,11 @@
     "fromTag": "use-vaults-auctions"
   },
   "scripts": {
-    "build": "yarn run build:sdk && yarn run build:submissions && yarn run build:synthetic-chain",
+    "build": "yarn run build:sdk && yarn run build:submissions && yarn run build:synthetic-chain && yarn run build:priceFeeds-for-mainnet && yarn run build:priceFeeds-for-devnet",
```

Comment:
I see only the 1 variant in the [ci job](https://github.com/Agoric/agoric-sdk/actions/runs/11265596360?pr=10146) Artifact:

```
$ unzip -l core-eval-scripts.zip |grep gov-price
     1024  2024-10-10 01:41   f-replace-price-feeds/gov-price-feeds-A3P_INTEGRATION-permit.json
     1463  2024-10-10 01:41   f-replace-price-feeds/gov-price-feeds-A3P_INTEGRATION-plan.json
     7069  2024-10-10 01:41   f-replace-price-feeds/gov-price-feeds-A3P_INTEGRATION.js
```

Is `build:submissions` the relevant thing to tweak rather than `build`?